compiler/llvm/clangUtil.cpp

```diff
@@ -2895,6 +2895,14 @@ static void helpComputeClangArgs(std::string& clangCC,
     clangCCArgs.push_back(clangRequiredWarningFlags[i]);
   }
 
+  if (usingGpuLocaleModel()) {
+    if (getGpuCodegenType() == GpuCodegenType::GPU_CG_NVIDIA_CUDA) {
+      // this is necessary for CUB 11. Once we drop CUDA 11 support, we can
+      // probably remove this
+      clangCCArgs.push_back("-Wno-deprecated-builtins");
+    }
+  }
+
```

Comment:
nit, this could just be `usingGpuLocaleModel() && getGpuCodegenType() == GpuCodegenType::GPU_CG_NVIDIA_CUDA`